Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/Approval/ApprovalTests.Verify_All.ForScenario.Profiles.Test_Controllers_Controllers.generated.approved.cs

```diff
@@ -16,6 +16,69 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 
+namespace EdFi.Ods.Api.Services.Controllers.AcademicWeeks.EdFi.Academic_Week_Readable_Excludes_Optional_References
```

Comment:
This new code is based on profile information added in `v6.1-patch1`, and it should have been added to the repo in that patch release.